i18n.install

```diff
@@ -50,13 +64,162 @@ function i18n_update_last_removed() {
 }
 
 /**
- * Move book settings from variables to config.
+ * Empty hook.
  */
 function i18n_update_1000() {
-  // Migrate variables to config.
-  // $config = config('i18n.settings');
-  // $config->set('book_allowed_types', update_variable_get('book_allowed_types', array('book')));
-  // $config->save();
-  // Delete variables.
-  // update_variable_del('book_allowed_types');.
+  // Got removed.
+}
+
+/**
+ * Disable unavailable and deprecated modules.
+ */
+function i18n_update_1001() {
+  // Neither can we, nor do we need to run any code, just rip out of the system
+  // table.
+  $obsolete = array(
+    'variable',
+    'variable_realm',
+    'variable_store',
+    'i18n_block',
+    'i18n_user',
+    'i18n_variable',
+  );
+  db_delete('system')->condition('name', $obsolete, 'IN')->execute();
+}
+
+/**
+ * Get translatables from variable_store and convert to locale.
+ */
+function i18n_update_1002() {
+  if (!db_table_exists('variable_store')) {
+    return;
+  }
+  // Hardcoded for now.
+  // @see https://github.com/backdrop/backdrop-issues/issues/4888
+  $mail_translatables = array(
```

Comment:
We can't predict *when* people will run this. I guess you're assuming this will be removed before core might add new translatable config to core and/or fix the core bug?

Reply:
> We can't predict when people will run this...

If i18n_update_dependencies() isn't enough (it probably isn't), we could additionally use update_already_performed() here.
Actually I'm hoping that https://github.com/backdrop/backdrop/pull/3504 gets merged into core soon enough, that I can revert this workaround. It's currently needed for proper testing - at least, I needed it to verify that this update hook works as expected.